pkg/pb/contact.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package contact;
+
+import "gogo.proto";
+
+service Contact {
+    rpc Ping(PingRequest) returns (PingResponse);
+    rpc PingBack(PingBackRequest) returns (PingBackResponse);
```

Comment:
Shouldn't these RPC-s be on different servers. Storage Node would only need `Pingback`, and Satellite would only need `Ping`.

nit: I think pingback a single word, so `Pingback`

Reply:
yeah good point. let me fix.

Reply:
@egonelbre I added that, but what do you think about the names of the 2 services?

Reply:
The package is already called `contact` so the full name would end up being `contact.ContactNode`.

Reply:
do you have any recommendations for names here?

Reply:
At the moment no good ones.

Reply:
how about ?
```
service TargetNode {
    rpc Pingback(PingbackRequest) returns (PingbackResponse);
}

service SenderNode {
    rpc Ping(PingRequest) returns (PingResponse);
}
```

Reply:
`Pinger`, `Pingee` :D... might make sense to do the naming exercise.

Reply:
Maybe names `Satellite` and `StorageNode` would make the most sense... however then I would like to avoid them attaching to the protobuf package.